buster/docparser.py

```diff
@@ -198,8 +199,11 @@ def generate_embeddings(
     "--embeddings-engine", default=EMBEDDING_MODEL, help=f"Embedding model to use. Default is {EMBEDDING_MODEL}"
 )
 def main(documents_csv: str, output_filepath: str, max_words: int, embeddings_engine: str):
+    # prepare the documents manager
+    documents_manager = DocumentsDB(output_filepath)
+
```

Comment:
wait wasn't this done in the previous PR ?

Reply:
yes it was, i think it didnt pick up the diff properly

Reply:
i merged main back in, i think its because i merged off the branch and then we did a squash merge.